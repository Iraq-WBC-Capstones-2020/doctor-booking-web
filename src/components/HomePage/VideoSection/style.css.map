{
    "version": 3,
    "mappings": "AACA,AAAA,aAAa,CAAC;EACd,UAAU,EAAE,IAAI;EAChB,aAAa,EAAE,IAAI;CAWlB;;AAbD,AAIA,aAJa,CAIb,EAAE,CAAA;EACE,UAAU,EAAE,KAAK;EACjB,KAAK,EAAE,KAAK;EACZ,UAAU,EAAE,MAAM;EAClB,WAAW,EAAE,GAAG;EAChB,SAAS,ECTA,IAAI;EDUb,WAAW,EAAE,IAAI;EACjB,KAAK,ECZE,OAAO;CDab;;AAGL,AAAA,IAAI,CAAA;EACA,gBAAgB,EAAE,OAAO;EACzB,WAAW,EAAE,MAAM;CACtB;;AAED,AAEI,WAFO,CAEP,GAAG,CAAA;EACC,KAAK,EAAE,QAAQ;EACf,MAAM,EAAC,KAAM;CAChB",
    "sources": [
        "style.scss",
        "__variables.scss"
    ],
    "names": [],
    "file": "style.css"
}