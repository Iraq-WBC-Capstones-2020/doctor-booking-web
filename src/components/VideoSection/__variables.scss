$pColor: #026fc2;
$hFontSize: 36px;
