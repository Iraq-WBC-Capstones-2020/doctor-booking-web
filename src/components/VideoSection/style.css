.videoSection {
  margin-top: 99px;
  margin-bottom: 99px;
}

.videoSection h1 {
  width: 350px;
  font-style: normal;
  font-weight: 500;
  font-size: 36px;
  line-height: 46px;
  color: #026fc2;
}

body {
  background-color: #fafbfd;
  font-family: Roboto;
}

.videoFrame img {
  width: 100%;
}
/*# sourceMappingURL=style.css.map */