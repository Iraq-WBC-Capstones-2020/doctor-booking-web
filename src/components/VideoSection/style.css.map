{
    "version": 3,
    "mappings": "AACA,AAAA,aAAa,CAAC;EACd,UAAU,EAAE,IAAI;EAChB,aAAa,EAAE,IAAI;CAElB;;AAED,AAAA,IAAI,CAAA;EACA,gBAAgB,EAAE,OAAO;EACzB,WAAW,EAAE,MAAM;CAEtB;;AACD,AACI,WADO,CACP,GAAG,CAAA;EACC,KAAK,EAAE,QAAQ;EACnB,MAAM,EAAC,KAAM;CACZ;;AAEL,AAAA,EAAE,CAAA;EACE,UAAU,EAAE,KAAK;EACjB,KAAK,EAAE,KAAK;EACZ,UAAU,EAAE,MAAM;EAClB,WAAW,EAAE,GAAG;EAChB,SAAS,ECtBA,IAAI;EDuBb,WAAW,EAAE,IAAI;EAGrB,KAAK,EC3BM,OAAO;CD4BjB",
    "sources": [
        "style.scss",
        "__variables.scss"
    ],
    "names": [],
    "file": "style.css"
}