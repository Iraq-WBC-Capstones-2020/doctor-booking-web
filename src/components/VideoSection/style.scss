@import './__variables.scss';
.videoSection {
margin-top: 99px;
margin-bottom: 99px;

h1{
    margin-top: 109px;
    width: 350px;
    font-style: normal;
    font-weight: 500;
    font-size: $hFontSize;
    line-height: 46px;
    color:$pColor;
    }
}

body{
    background-color: #FAFBFD;
    font-family: Roboto;
}

.videoFrame{
    
    img{
        width: 450.43px;
        height:313px ;
    }
}

