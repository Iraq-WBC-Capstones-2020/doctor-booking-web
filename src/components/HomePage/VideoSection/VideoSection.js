import React from 'react';
import { Container, Row } from 'react-bootstrap';
import VideoFrame from './images/Video.png';
import './style.css';

function VideoSection() {
  return (
    <Container id="videoSection" className="videoSection">
      <Row>
        <div className="col-lg-6">
          <h1>
            To get more familiar <br />
            with the website
            <br />
            watch this video
            <br />
          </h1>
        </div>

        <div className="col-lg-6">
          <div className="videoFrame">
            <img alt="Watch this video" src={VideoFrame} />
          </div>
        </div>
      </Row>
    </Container>
  );
}
export default VideoSection;
