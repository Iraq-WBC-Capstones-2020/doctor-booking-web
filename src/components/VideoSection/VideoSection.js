import React from 'react';
import { Container, Row } from 'react-bootstrap';
import videoFrame from './images/Video.png';
import './style.css';

function VideoSection() {
  return (
    <Container className="VideoSection">
      <Row>
        <div className="col-lg-6">
          <h1>
            To get more familiar <br />
            with the website
            <br />
            watch this video
            <br />
          </h1>
        </div>

        <div className="col-lg-6">
          <div className="videoFrame">
            <img alt="Watch this video" src={videoFrame} />
          </div>
        </div>
      </Row>
    </Container>
  );
}
export default VideoSection;
