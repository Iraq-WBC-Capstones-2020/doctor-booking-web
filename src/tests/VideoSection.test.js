import React from 'react';
import { render } from '@testing-library/react';
import VideoSection from '../components/VideoSection/VideoSection';
import renderer from 'react-test-renderer';
it('render1', () => {
  render(<VideoSection />);
});

it('renders correctly', () => {
  const tree = renderer.create(<VideoSection />).toJSON();
  expect(tree).toMatchSnapshot();
});
