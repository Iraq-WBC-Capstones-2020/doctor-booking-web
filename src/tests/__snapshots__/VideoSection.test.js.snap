// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="VideoSection container"
>
  <div
    className="row"
  >
    <div
      className="col-lg-6"
    >
      <h1>
        To get more familiar 
        <br />
        with the website
        <br />
        watch this video
        <br />
      </h1>
    </div>
    <div
      className="col-lg-6"
    >
      <div
        className="videoFrame"
      >
        <img
          alt="Watch this video"
          src="Video.png"
        />
      </div>
    </div>
  </div>
</div>
`;
